src/easypay/payment.py

```
@@ -47,6 +54,8 @@ def generate_payment(
         method["currency"] = currency
         method["warning"] = warning
         method["cancel"] = cancel
+        if not customer == None:
```

Ruff (0.8.2)

Use `customer != None` instead of `not customer == None`

Replace with `!=` operator

(SIM201)